app/pages/sections/commencement-date/contextCreator.js

```diff
@@ -1,9 +1,10 @@
 import manifest from './manifest.json';
 import { baseUrl } from '../../../config';
+import { formatCommencementDate } from '../../../helpers/dateFormatter';
 
 const addDataToManifest = (commencementDate) => {
   if (commencementDate) {
-    const [year, month, day] = commencementDate.split('-');
+    const [day, month, year] = formatCommencementDate(commencementDate).split(' ');
```

Comment:
might as well return the array from this function `const [day, month, year] = formatCommencementDate(commencementDate)`